### PR TITLE
Enhance debug output and readability

### PR DESCRIPTION
**Pull request by apostasie**
Small quality-of-life improvements.

1. `--quiet` ancillary image pull
2. RootFul | RootLess -> Rootful, Rootless (previous discussion in part 4)
3. test title tweak

Forked out of #3535 

@AkihiroSuda PTAL

### CODE REVIEW

File: pkg/testutil/nerdtest/requirements.go [hunk 1]
```diff
@@ -145,8 +145,8 @@ var RootLess = &test.Requirement{
 	},
 }
 
-// RootFul marks a test as suitable only for rootful env
-var RootFul = test.Not(RootLess)
+// Rootful marks a test as suitable only for rootful env
+var Rootful = test.Not(Rootless)
```
Comment (AkihiroSuda) [inline pkg/testutil/nerdtest/requirements.go:149]: I expect this to be separate commit, as it doesn’t relate to --quiet

Comment (apostasie) [reply pkg/testutil/nerdtest/requirements.go:149]: Done.

**Review by AkihiroSuda — APPROVED** [review]
Thanks
